src/aimes/emgr/emgr.py

```diff
@@ -309,6 +309,41 @@ def derive_pilot_descriptions(cfg, strategy):
 
 
 # -----------------------------------------------------------------------------
+def derive_pilot_descriptions_osg(cfg, strategy):
+
+    #   Initializing list used to hold pilot descriptions
+    pdescs = list()
+
+    #   Creating the specified number of OSG pilots. Each pilot is given
+    #   the project ID, resource, number of cores, and wall time
+    for pilot in range(cfg['num_osg_pilots']):
+
+        pdesc = rp.ComputePilotDescription()
+
+        pdesc.project = cfg['project_ids'].get('osg.xsede-virt-clust')
+        pdesc.resource = 'osg.xsede-virt-clust'
+
+        print "DEBUG: strategy['inference'] = %s" % strategy['inference']
+
+        pdesc.cores = 1
```

Comment:
I would probably move also this to a configuration parameter. The rationale: we may want to test what happens when we ask for 2 cores per pilot on OSG--e.g., super-long Tq? No resources available?